### ISSUE TITLE
Killer feature: Support for nextcloud Flow

### ISSUE THREAD

**Issue by marcelklehr** [kffl/nextcloud-webhooks#1]
Thank you for creating this app!

The functionality this app provides is an often sought for request for existing apps. Allowing users to setup custom webhooks with nextcloud flow would allow seamless integration of this app with a lot of apps.

**Comment by kffl** [comment]
Thank you for your feedback!

While researching the possibility of integrating this app with Nextcloud Flow, I have found the following information in the [release notes](https://nextcloud.com/blog/nextcloud-hub-20-debuts-dashboard-unifies-search-and-notifications-integrates-with-other-technologies/) of Nextcloud 20:

> Flow adds push notification and webhooks so other web apps can easily integrate with Nextcloud

Despite that, I was unable to find any webhooks-like functionality that was introduced in v20. Some [other users](https://help.nextcloud.com/t/flow-adds-push-notification-and-webhooks-so-other-web-apps-can-easily-integrate-with-nextcloud/98574) also appear to be confused by that release note. After a bit more digging, I have found the following repo:

https://github.com/nextcloud/flow_webhooks

Dates of the last non-bot commits 
apeear to
